var/spack/repos/builtin/packages/pfunit/package.py

```diff
@@ -60,6 +71,14 @@ def cmake_args(self):
             '-DOPENMP=%s' % ('YES' if '+openmp' in spec else 'NO'),
             '-DMAX_RANK=%s' % spec.variants['max_array_rank'].value]
 
+        if spec.satisfies('@3.2.9 %xl_r'):
+            # DCMAKE_Fortran_MODULE_DIRECTORY setting is
+            # not sufficient for XLF build, at least on rzansel
+            # need to create the path so module files can be placed
+            # appropriately
+            pathlib.Path(spec.prefix.include).mkdir(parents=True, exist_ok=True)
```

Comment:
```suggestion
            mkdir(self.prefix.include)
```
This will fix Python 2.6 support